.travis.yml

```diff
@@ -39,4 +39,5 @@ deploy:
   file_glob: true
   on:
     tags: true
-    repo: appsody/appsody
\ No newline at end of file
+    repo: appsody/appsody
```

Comment:
I only see whitespace changes in the `.travis.yml` file, please try to remove these from the PR

Reply:
Agreed.